src/app/components/quotes/quotes.hbs

```diff
@@ -1,36 +1,36 @@
 <style scoped>
-    @import url(/app/pages/home/quotes/quotes.css);
+    @import url(/app/components/quotes/quotes.css);
 </style>
 <div class="quotes">
     <div class="quote-bucket right">
-        <img src="/images/home/quotes/juan.jpg" width="175" height="230" />
+        <div class="image juan"></div>
```

Comment:
I've been contemplating how to do this, since it's going to come from the server.  Ideally we'd be able to use `data-image="/images/home/quotes/juan.jpg"`, but no browser currently supports the CSS `attr()` function (which would allow us to do: `background-image: attr(data-image url)`).

So I think in the meantime our only option will be to define classes in CSS for each image in the quotes SCSS file, and then have the template add the class dynamically (probably by just basing it on the file name).

We could obviously just inject the image src if it wasn't a background-image, but, for design reasons, it needs to be.